### ISSUE TITLE
antimeridian.bbox possibly returning wrong result

### ISSUE THREAD

**Issue by jessjaco** [gadomski/antimeridian#134]
I have a set of features in the attached file which are on both sides the antimeridian, but do not span it. However, `antimeridian.bbox` appears to be returning their values in the wrong order. Or, is the the desired behavior, as none of the features cross the antimeridian itself?

```python
import antimeridian
import geopands
antimeridian.bbox(antimeridian.fix_multi_polygon(geopandas.read_file('tricky.zip').geometry.loc[0]))
```
produces:
`[-179.92643501801794, -19.044135782844712, 179.97098076944076, -18.555752850452095]
`

[tricky.zip](https://github.com/user-attachments/files/16822042/tricky.zip)

**Comment by gadomski** [comment]
Looks like a bug, thanks for reporting!

**Comment by gadomski** [comment]
After digging in, it looks like `tricky.zip` was not being picked up as antimeridian-crossing because it doesn't have any points coincident to the antimeridian:

![tricky](https://github.com/user-attachments/assets/ce8a27ce-f1de-4305-9709-1b589689bb35)

I could implement a heuristic to guess that a given set of polygons are "close enough" to the antimeridian to consider the multipolygon antimeridian crossing, but I'm not _quite_ sure what that would be. I'll think about it.

**Comment by jessjaco** [comment]
Do you see the shapes on the other side (at about -19, -179.85)? That was why I considered the dataset to be crossing. My naive thought was that two bounding boxes could be calculated: one that crossed -180 and one that  crossed 0 and the smaller one could be used.

**Comment by gadomski** [comment]
> Do you see the shapes on the other side (at about -19, -179.85)?

I did.

> two bounding boxes could be calculated: one that crossed -180 and one that crossed 0 and the smaller one could be used.

Yup, that's no a bad heuristic but it _could_ be wrong, e.g. for a bbox that covers New Zealand, Madagascar, and Argentina.

I'm thinking we need more info from the caller, so maybe a `force_over_antimeridian` argument is the answer, which will calculate the antimeridian-crossing bbox no matter what.

**Comment by gadomski** [comment]
@jessjaco I added a `force_over_antimeridian` argument to `bbox`, available in v0.3.9: https://github.com/gadomski/antimeridian/releases/tag/v0.3.9
